packages/ai/core/middleware/default-settings-middleware.test.ts

```diff
@@ -41,6 +41,20 @@ describe('defaultSettingsMiddleware', () => {
       expect(result.temperature).toBe(0.5);
     });
 
+    // https://github.com/vercel/ai/pull/5769
```

Comment:
pls rm (to keep overhead low) - the tests should be the source of truth / specification

Reply:
```suggestion
```